.bazelrc

```diff
@@ -24,8 +24,9 @@ build:vm --define=folly_so=1
 build:vm --config=specify_vm_cc
 
 # MAGMA-BUILDER DOCKER CONTAINER CONFIGS
-build:docker --disk_cache=/magma/.bazel-cache
-common:docker --repository_cache=/magma/.bazel-cache-repo
+build:docker --disk_cache=/workspaces/magma/.bazel-cache
+common:docker --repository_cache=/workspaces/magma/.bazel-cache-repo
+build:docker --define=folly_so=1
```

Comment:
Is this needed in bazel-base?

Reply:
Yes, before https://github.com/magma/magma/pull/12638 Bazel Base builds did not use the folly-so, but the Devcontainer did, i.e., this needed to be aligned. I preferred the other way around, but this made problems with the github caches. This should be revised at some point - best would be paired with another approach to remove folly (this was tried before, but was not possible at that point).